skimage/filters/edges.py

```diff
@@ -706,27 +712,41 @@ def laplace(image, ksize=3, mask=None):
     return _mask_filter_result(result, mask)
 
 
-def farid(image, *, mask=None):
-    """Find the edge magnitude using the Farid transform.
+def farid(image, mask=None, *, axis=None, mode='reflect', cval=0.0):
+    """Find the edge magnitude using the Scharr transform.
```

Comment:
```suggestion
    """Find the edge magnitude using the Farid transform.
```

I guess a copy-paste gone astray?